yarn.lock

```diff
@@ -38262,7 +38262,6 @@ fsevents@~2.1.1:
     lodash.pick: 4.4.0
     luxon: ^2.5.2
     mailparser: 0.6.1
-    memcached: ^2.2.2

```

Comment:
We should also remove the script that starts memcached, the pm2 config that calls the script, and whatever other references are still in the repo.

Reply:
Let's also remove the memcached image from the circleci's config.yml.

Reply:
I can't yet remove memcache since it is used in customs server for backwards compatiabilty. Filed https://mozilla-hub.atlassian.net/browse/FXA-9356 to track this.